packages/mds-api-server/index.ts

```diff
@@ -41,99 +41,91 @@ const about = () => {
   }
 }
 
-interface ApiServerOptions {
-  authorizer: ApiAuthorizer
-  handleCors: boolean
+export const RequestLoggingMiddleware = () =>
```

Comment:
No functional changes to this module. Just exporting some reusable middleware and request handlers.